dag/minerals.yml

```diff
@@ -0,0 +1,61 @@
+steps:
+  #
+  # IEA - Critical minerals.
+  #
+  data://meadow/iea/2024-07-04/critical_minerals:
+    - snapshot://iea/2024-07-04/critical_minerals.xlsx
+  #
+  # IEA - Critical minerals.
+  #
+  data://garden/iea/2024-07-04/critical_minerals:
+    - data://meadow/iea/2024-07-04/critical_minerals
+  #
+  # BGS - World Mineral Statistics.
+  #
+  data://meadow/bgs/2024-07-09/world_mineral_statistics:
+    - snapshot://bgs/2024-07-09/world_mineral_statistics.zip
+  #
+  # BGS - World Mineral Statistics.
+  #
+  data://garden/bgs/2024-07-09/world_mineral_statistics:
+    - data://meadow/bgs/2024-07-09/world_mineral_statistics
+    - data://garden/regions/2023-01-01/regions
+    - data://garden/wb/2024-03-11/income_groups
+  #
+  # USGS - Historical statistics for mineral and material commodities.
+  #
+  data://meadow/usgs/2024-07-15/historical_statistics_for_mineral_and_material_commodities:
+    - snapshot://usgs/2024-07-15/historical_statistics_for_mineral_and_material_commodities.zip
+  #
+  # USGS - Historical statistics for mineral and material commodities.
+  #
+  data://garden/usgs/2024-07-15/historical_statistics_for_mineral_and_material_commodities:
+    - data://meadow/usgs/2024-07-15/historical_statistics_for_mineral_and_material_commodities
+  #
+  # USGS - Mineral commodity summaries.
+  #
+  # NOTE: There is no meadow step, as processing and harmonization are tightly coupled.
+  data://garden/usgs/2024-07-15/mineral_commodity_summaries:
+    - snapshot://usgs/2024-07-15/mineral_commodity_summaries_2022.zip
+    - snapshot://usgs/2024-07-15/mineral_commodity_summaries_2023.zip
+    - snapshot://usgs/2024-07-15/mineral_commodity_summaries_2024.zip
+  #
+  # Minerals - Minerals.
+  #
+  # Compilation of minerals data from different origins.
+  data://garden/minerals/2024-07-15/minerals:
+    - data://garden/bgs/2024-07-09/world_mineral_statistics
+    - data://garden/usgs/2024-07-15/historical_statistics_for_mineral_and_material_commodities
+    - data://garden/usgs/2024-07-15/mineral_commodity_summaries
+  #
+  # Minerals - Minerals.
+  #
+  data://grapher/minerals/2024-07-15/minerals:
+    - data://garden/minerals/2024-07-15/minerals
+  #
+  # Minerals - Minerals explorer.
+  #
+  data://explorers/minerals/latest/minerals:
+    # TODO: The following would be the right dependency, but it fails because GrapherStep.checksum_output() is not implemented.
```

Comment:
Why is `data://grapher` step not sufficient?

Reply:
Hmmm. Just to be able to execute the steps, the `data://grapher` dependency will work. But, if you don't execute the `grapher://grapher` step, then you won't see the changes in the admin.
However, given that the `--explorer` flag also forces `--grapher`, then maybe the dependency on the `data://grapher` step is enough. I'll remove the TODO.